### PR TITLE
Add JetBrains Space Icon

### LINKED ISSUE #8795: Icon request: Jetbrains Space
### Brand Name

Space

### Website

https://www.jetbrains.com/space/

### Popularity Metric

The similarweb rank is 4,941. See https://www.similarweb.com/website/jetbrains.com/

### Official Resources for Icon and Color

Brand guideline:
* https://www.jetbrains.com/company/brand/

SVG files:
* https://resources.jetbrains.com/storage/products/company/brand/logos/Space.svg
* https://resources.jetbrains.com/storage/products/company/brand/logos/Space_icon.svg

### Additional Comments

Space is a complete development/collaboration platform from JetBrains, integrating with their IDEs.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by RiteshK-611**
**Issue:** closes #8795

**Similarweb rank:**
  
The similarweb rank is 4,941. See https://www.similarweb.com/website/jetbrains.com/

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
This change resolves issue #8795 by adding JetBrains Space logo in jetbrainsspace.svg file and by updating simple-icons.json file with icon's info

### Preview
jetbrainsspace

### CODE REVIEW

**Comment by PratyushJayachandran** [comment]
Did you pick this SVG from their website?
Just asking because I see a different one on their website. An invert of what you added.

**Comment by LitoMore** [comment]
Could you two @RiteshK-611 @PratyushJayachandran point out where did you find your icons? I can't find the flat version of icon on their website. Thanks.

**Comment by PratyushJayachandran** [comment]
> Could you two @RiteshK-611 @PratyushJayachandran point out where did you find your icons? I can't find the flat version of icon on their website. Thanks.

https://www.jetbrains.com/fleet/
Keep scrolling down and you will see them mentioning space. They also have a branding page with all icons in color as svg and png: https://www.jetbrains.com/company/brand/#logos-and-icons

**Comment by LitoMore** [comment]
We should always pick resource from their [Brand Assets](https://www.jetbrains.com/company/brand/) page. Or asking them for an approval to use the informal flat icon.

image

**Comment by PratyushJayachandran** [comment]
> We should always pick resource from their [Brand Assets](https://www.jetbrains.com/company/brand/) page. Or asking them for an approval to use the informal flat icon.

I see this problem with almost all brands. They always say don't alter colors and it makes sense too. But at really small sizes like the badge, we would like it to be flat.

Is providing logos in original design something that is planned for future?

**Comment by LitoMore** [comment]
> Is providing logos in original design something that is planned for future?

I don't think so.

We can suggest JetBrains team make a new flat icon for badges.

**Comment by RiteshK-611** [comment]
> Did you pick this SVG from their website? Just asking because I see a different one on their website. An invert of what you added. image

Actually I alter the original SVG to make it flat, but now I updated it with the above one which looks better
Thanks @PratyushJayachandran for this

**Comment by PratyushJayachandran** [comment]
> We can suggest JetBrains team make a new flat icon for badges.
They might actually agree too but for other companies who don't know what this project is for and how it is used, do we have some template that we can use to ask the brands? 

From what I see saying yes to an exception means exception for everyone else too. Like some companies explicitly mention how to use logo when you want it flat. Some companies say that you may use it freely for editorial work and things like that. If someone can chime in and tell what is the language one should use, that would be great. We can include it in our contribution guides.

**Comment by adamrusted** [comment]
We have set a precedent with other JetBrains products to use the full icon (minus the background squiggles) as our icon. We should probably continue that here.

**Comment by adamrusted** [comment]
In an interesting turn of events - JetBrains have kept the Space logo previously contributed!

image

I'll review this one shortly.

File: icons/jetbrainsspace.svg
Comment (adamrusted) [inline icons/jetbrainsspace.svg:1]: image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Spotting a slight difference between your icon and my re-rendered version with a 1px gap between each section. Could you have another look please @RiteshK-611?

**Comment by PeterShaggyNoble** [comment]
Closing due to inactivity. If you would like to continue working on this, @RiteshK-611 then we can reopen it for you. If someone else would like to take it on then please open a new PR.

**Pull request closed without merging**